src/main/java/de/komoot/photon/query/.gitignore

```diff
@@ -0,0 +1 @@
+/PhotonQueryBuilderBackup.java
```

Comment:
Putting java files in gitignore seems wrong

Reply:
Hello, I deleted the .gitignore file. 
Thank you very much!
David